repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -179,20 +177,20 @@ def dnf_shell_rpm_download(self, overlayfs_info):
             script.write('\n'.join(cmds))
             script.flush()
 
-        error = preparetransaction.check_container_call(overlayfs_info, [
-            '--',
-            '/bin/bash',
-            '-c',
-            ' '.join(dnf_command + ['/var/lib/leapp/dnf-script.txt'])])
+        # FIXME: fails on insufficient permissions
+        cmd = ['--', '/bin/bash', '-c', ' '.join(dnf_command + ['/var/lib/leapp/dnf-script.txt'])]
+        _, error = preparetransaction.guard_container_call(overlayfs_info, cmd, checkers=(preparetransaction.connection_checker, preparetransaction.space_checker))
 
         if os.environ.get('LEAPP_DEBUG', '0') == '1':
             # We want the debug data available where we would expect it usually.
+            # FIXME: no exception is caught, retcode is not checked
             call(['rm', '-rf', os.path.join('/tmp', 'download-debugdata')])
             call(['cp', '-a', os.path.join(overlayfs_info.merged, 'debugdata'), '/tmp/download-debugdata'])
```

Comment:
That might be rather the one we care about failing, but we shouldn't cause it to break the upgrade - since this is debug data only